index.js

```diff
@@ -85,7 +88,8 @@ module.exports = (config) => {
 	event.dispatcher.on(event.test.started, async (test) => {
 		if (test.body) {
 			if (test.body.includes('addExampleInTable')) {
-				const testRailTag = /"testRailTag":"(@C\d+)"/.exec(test.title);
+				const testRailTagRegExp = new RegExp(`"testRailTag":"(${prefixRegExp}")`)
```

Comment:
It doesn't work.
Example of `test.title`: 
`Some test {"testRailTag":"@C12345","someParam":"Some param value"}`

Your new default pattern(value of `testRailTagRegExp`):
`"testRailTag":"(\/@C\d+\/")`


@wjaros fix the changes, it can't match.

@PeterNgTr FYI